.github/workflows/python_collect_docstrings.yml

```diff
@@ -0,0 +1,39 @@
+name: pyfiction Collect Docstrings
+
+on: workflow_dispatch
+
+jobs:  
+  get_docstrings:    
+    name: collect docstrings    
+    runs-on: macos-latest  
```

Comment:
Can we run the action on `ubuntu-latest` instead? Usually, there are more Ubuntu runners available, which gives this workflow a better response time